x-pack/plugins/observability_solution/inventory/public/hooks/use_theme.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEuiTheme } from '@elastic/eui';
+import { useKibana } from './use_kibana';
+
+export function useTheme() {
```

Comment:
Is this really needed? You can use `useEuiTheme`  from `@elastic/eui`

Reply:
You mean this part ? 

```
euiTheme: useEuiTheme().euiTheme
```

Reply:
I mean not have this custom hook at all.

Reply:
you're right. useEuiTheme provides that with `colorMode`

updating it

Reply:
https://github.com/elastic/kibana/pull/193557/commits/dbc1f80ca8dbbd7d9fc1601f22677932cd9d8632